.circleci/config.yml

```diff
@@ -342,6 +342,31 @@ jobs:
       - *addsshkeys
       - *commitworkstationdebs
 
+  build-bullseye-securedrop-log:
+    docker:
+      - image: circleci/python:3.9-bullseye
+    steps:
+      - checkout
+      - *removevirtualenv
+      - *installdeps
+      - *clonesecuredroplog
+      - *getnightlyversion
+      - *builddebianpackage
+
+  build-nightly-bullseye-securedrop-log:
+    docker:
+      - image: circleci/python:3.7-buster
```

Comment:
@eaon @legoktm - This looks to be the wrong image. May be a typo, and could explain why it's not in nightlies (see https://apt-test.freedom.press/pool/nightlies/s/securedrop-log/)